lib/timescaledb/model/peer_to_room.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Membrane.RTC.Engine.TimescaleDB.Model.PeerToRoom do
+  @moduledoc """
+  Model representing relation between room and its peers in `Membrane.RTC.Engine`.
+  """
```

Comment:
This isn't the way such relations should be defined in Ecto. Please read the docs on `has_many`, `has_one` and `many_to_many` from https://hexdocs.pm/ecto/Ecto.Schema.html#has_many/3